.github/workflows/lualint.yml

```
@@ -0,0 +1,61 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Lint changed-files
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v35
+
+      - name: Generate token
+        id: github-app-token
+        uses: tibdex/github-app-token@021a2405c7f990db57f5eae5397423dcc554159c # v1.7.0
+        with:
+          app_id: ${{ vars.KONG_GH_APP_ID }}
+          private_key: ${{ secrets.KONG_GH_APP_PRIVATE_KEY }}
+
+      - name: Download Lualint
+        uses: robinraju/release-downloader@v1.7
+        with:
+          repository: "kong/lualint"
+          token: ${{ steps.github-app-token.outputs.token }}
+          latest: true
+          fileName: "lualint_x86_64-unknown-linux-gnu.zip"
+
+      - name: Lint all changed lua files
+        run: |
+          unzip lualint_x86_64-unknown-linux-gnu.zip;
+          chmod +x ./lualint;
+          for file in ${{steps.changed-files.outputs.all_changed_files}}; do
```

```suggestion
          for file in ${{ steps.changed-files.outputs.all_changed_files }}; do
```

We should be consistent over styles of above.